sherpa-onnx/csrc/online-transducer-decoder.h

```diff
@@ -82,6 +82,11 @@ class OnlineTransducerDecoder {
   virtual void Decode(Ort::Value encoder_out,
                       std::vector<OnlineTransducerDecoderResult> *result) = 0;
 
+  virtual std::vector<Ort::Value> Decode_me(Ort::Value encoder_out,
```

Comment:
Your compilation error is caused by this method.

Please remove it.

Also, your greedy search decoder does not need to inherit from this class.